src/resources/mathjax_preview_template.html

```diff
@@ -10,7 +10,7 @@
     <link rel="stylesheet" type="text/css" href="CSS_PLACE_HOLDER">
     <script src="qrc:/resources/qwebchannel.js"></script>
     <!-- EXTRA_PLACE_HOLDER -->
-    <script src="JS_PLACE_HOLDER" async></script>
+    <script src="JS_PLACE_HOLDER" async id="MathJax-script"></script>
```

Comment:
What is the purpose of this id?

Reply:
MathJax v3 refers the `<script>` itself because MathJaX loads some plugin dynamically by need, e.g., fonts, stylesheet. See also, http://docs.mathjax.org/en/latest/web/start.html#using-mathjax-from-a-content-delivery-network-cdn